src/main/java/address/controller/TagEditDialogController.java

```diff
@@ -1,23 +1,23 @@
 package address.controller;
 
 import address.events.EventManager;
-import address.model.datatypes.ContactGroup;
+import address.model.datatypes.Tag;
 import javafx.fxml.FXML;
 import javafx.scene.control.Alert;
 import javafx.scene.control.TextField;
 
-public class GroupEditDialogController extends EditDialogController {
+public class TagEditDialogController extends EditDialogController {
     @FXML
-    private TextField groupNameField;
+    private TextField tagNameField;
 
-    private ContactGroup finalGroup;
+    private Tag finalTag;
```

Comment:
I think editedTag may be a better name. I guess the method getFinalInput could be changed also?